src/main/scala/io/treeverse/clients/LakeFSInputFormat.scala

```diff
@@ -12,10 +12,12 @@ import org.apache.hadoop.mapreduce._
 import java.io.{DataInput, DataOutput, File}
 import scala.collection.JavaConverters._
 
-class GravelerSplit(var range: RangeData, var path: Path) extends InputSplit with Writable {
+class GravelerSplit(var range: RangeData, var path: Path)
+    extends InputSplit
+    with Writable {
   def this() = this(null, null)
 
-  override def write(out: DataOutput) = {
+  override def write(out: DataOutput): Unit = {
```

Comment:
Cool, this is actually [recommended by the style guide](https://docs.scala-lang.org/style/declarations.html#procedure-syntax).  It is also _different_ from what I originally wrote: omitting the return value _deduces_ the return type, so it changes "invisibly" as the method changes :-/